src/components/logo/logo.tsx

```diff
@@ -74,6 +75,14 @@ class Logo extends Component<any, any> {
     }
   }
 
+  _handleKeyDown = (e: KeyboardEvent): void => {
+    if (e.keyCode === KeyMap.ENTER || e.keyCode === KeyMap.SPACE) {
+      window.open(this.state.urlLink, '_blank');
```

Comment:
what if we are in an iframe ?

Reply:
the solution works fine, example from ui-managers
https://github.com/kaltura/playkit-js-ui-managers/blob/3b2567a5093ae94b11b7bc38c5ab08b4fcf2bd37/src/services/side-panels-manager/models/item-wrapper.tsx#L81